src/test/java/org/mifos/integrationtest/cucumber/stepdef/VoucherManagementStepDef.java

```diff
@@ -313,11 +321,7 @@ public void createVoucher() {
         mockServerStepDef.startStub("/createVoucher", PUT, 200);
         mockServerStepDef.startStub("/activateVoucher", PUT, 200);
         iCallTheVoucherCreateAPIWithExpectedStatusOf(202, "/createVoucher");
-        try {
-            Thread.sleep(5000);
-        } catch (InterruptedException e) {
-            throw new RuntimeException(e);
-        }
+        await().atMost(awaitMost, SECONDS).pollInterval(pollInterval, SECONDS).until(() -> true);
```

Comment:
we dont need this sleep here, add it inside iShouldBeAbleToExtractResponseBodyFromCallback() instead